docs: correct minor grammatical error in documentation (#478)
aica-technology/api: docs/core/examples/guides/camera-streamer.md

Once you have selected an appropriate **source**:

1. Press **Start** to start the application.

> 2. To see the live camera feed, click on the Launcher tab on the upper right corner and select **Launch RViz**.

Reviewer: ```suggestion
2. To see the live camera feed, select **Launch RViz** from the Launcher settings
```
Your change is correct with the "upper right corner" but since this depends on the version of Studio and Launcher you have, I would rather go more general without specifying the location of this option. This is also how we have done it [here](https://docs.aica.tech/core/examples/guides/yolo-example#running-the-application)
